src/rules.jl

```diff
@@ -150,6 +150,12 @@ end
 @define_diffrule SpecialFunctions.bessely1(x)    =
     :(  (SpecialFunctions.bessely0($x) - SpecialFunctions.bessely(2, $x)) / 2 )
 
+# logerfcx added in SpeciaFunctions 0.10, which requires Julia 1.3
+if VERSION ≥ v"1.3"
+    @define_diffrule SpecialFunctions.logerfcx(x) =
+        :(  2 * ($x - inv(SpecialFunctions.erfcx($x)) / sqrt(π))  )
```

Comment:
I suggest to use either
```suggestion
        :(  2 * ($x - SpecialFunctions.erfcx(- $x) / sqrt(π))  )
```
or
```suggestion
        :(  2 * ($x - inv(SpecialFunctions.erfcx($x) * sqrt(π)))  )
```

Reply:
I think the first suggestion is wrong, did you test it?

Here I just copied the rule defined in ChainRules, which seems to work well.
So, do you have any reason to prefer the second suggestion?

Reply:
It should be correct, it's how the rule is defined in SpecialFunctions. The rules were moved from ChainRules to SpecialFunctions quite a while ago.

Reply:
Ah never mind, I missed that it's exp and not erfcx: https://github.com/JuliaMath/SpecialFunctions.jl/blob/1febdd05902f5b3a3adc9915aba40b776aa3ea24/src/chainrules.jl#L44

Reply:
The second suggestion combines the two divisions, so to me it seemed a bit simpler.

Reply:
Ok I added your second suggestion.